moorhuhn-service/src/main/java/com/moorhuhnservice/moorhuhnservice/controller/MoorhuhnController.java

```diff
@@ -0,0 +1,43 @@
+package com.moorhuhnservice.moorhuhnservice.controller;
+
+import com.moorhuhnservice.moorhuhnservice.BaseClasses.Question;
+import com.moorhuhnservice.moorhuhnservice.repositories.QuestionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class MoorhuhnController {
+
+    @Autowired
+    QuestionRepository questionRepository;
+
+    @PostMapping("/saveAllQuestions")
+    public List<Question> saveAllQuestions(@RequestBody List<Question> questions) {
+        System.out.println("try to save all "+questions.size()+" questions: ");
+        return questions;
+    }
+
+    @PostMapping("/saveFirstTestQuestion")
+    public Question saveFirstTestQuestion() {
+        System.out.println("try to save a static question");
+        Question questionTest = new Question("xyz", "Frage1","AntwortRichtig","AntwortFalsch1","AntwortFalsch2","AntwortFalsch3","AntwortFalsch4");
+        questionRepository.save(questionTest);
+        return questionTest;
+    }
+
+    @PostMapping("/saveSecondTestQuestion")
+    public Question saveSecondTestQuestion() {
+        System.out.println("try to save a static question");
+        Question questionTest = new Question("xyz", "Frage2","AntwortRichtig","AntwortFalsch1","AntwortFalsch2","AntwortFalsch3","AntwortFalsch4");
+        questionRepository.save(questionTest);
+        return questionTest;
+    }
+
+    @GetMapping("/getAllQuestions")

```

Comment:
URLs should not be PascalCased. They should be kebab-cased instead.
Also, I'm not quite sure why some of these routes are needed.
Or why you use `System.out.println` instead of the Spring native logger...

Reply:
2 of this are only for Test purposes, therefore it stands "TEST" inside the route, the other post method is not functionally yet, therefore you can ignore that. i will change the route for the get method.